.shopify-build/krane.yml

```diff
@@ -6,16 +6,15 @@ steps:
   - label: Lint
     timeout: 5m
     run:
+      - bundle: ~
```

Comment:
what does this syntax do?

Reply:
I believe it says run bundler and cache the results. I followed the instructions at the bottom of https://shopify-build.docs.shopify.io/troubleshooting/deprecations/